glow_old.py

```diff
@@ -224,11 +224,10 @@ def infer(self, spect, sigma=1.0):
 
     @staticmethod
     def remove_weightnorm(model):
-        waveglow = copy.deepcopy(model)
+        waveglow = model
         for WN in waveglow.WN:
             WN.start = torch.nn.utils.remove_weight_norm(WN.start)
             WN.in_layers = remove(WN.in_layers)
             WN.cond_layers = remove(WN.cond_layers)
-            WN.res_layers = remove(WN.res_layers)
-            WN.skip_layers = remove(WN.skip_layers)
+            WN.res_skip_layers = remove(WN.res_skip_layers)
```

Comment:
let's keep res_layers and skip_layers and the deep copy

Reply:
The remove_weightnorm function call is after model conversion. The conversion function removes res_layers and skip_layers and adds res_skip_layers.

Reply:
we're not converting the model automatically and the pre-trained model loads from glow_old.py which has res_layers and keep_layers.

Reply:
The pre-trained model does not work with inference.py out of the box. It complains res_skip_layers not found if I do not change glow_old.py. It is somehow loading from glow.py and not glow_old.py

Reply:
Also the converted model also fails to load with the same error if I do not change glow_old.py

Reply:
The old model fails to load either way: if I change or do not change glow_old.py